src/utils/mtev_log.c

```diff
@@ -399,6 +399,11 @@ int mtev_log_global_enabled(void) {
   return LIBMTEV_LOG_ENABLED();
 }
 
+mtev_boolean mtev_log_has_material_output(mtev_log_stream_t ls) {
+  /* This is materialized in already into flags_below as "ENABLED" */
```

Comment:
Think this comment has typos in it - not clear

Reply:
fixed